samples/web/content/testrtc/js/gumhandler.js

```diff
@@ -42,7 +47,7 @@ GumHandler.prototype = {
     }
     this.gumPendingDialog_.close();
     this.gumErrorDialog_.close();
-    startButton.removeAttribute('disabled');
+    this.gumStreamSuccessCallback();
```

Comment:
suffix with _ and set it to null on the constructor.. as that allows to easily see all members used by this class.